### PR TITLE
[Feature] Text rendering + improved box

### PR DESCRIPTION
**Pull request by 64blit**
- Added simple box rendering + improved corner rendering + labels that fit in bounds better
- Text rendering that scales to height and width

image

image

### CODE REVIEW

File: src/eyepop-render-2d/render-box.ts [hunk 1]
```diff
@@ -21,16 +21,18 @@ export class RenderBox implements Render
     private showNestedClasses: boolean
     private showConfidence: boolean
     private showTraceId: boolean
+    private boxType?: string
 
     constructor(options: Partial<RenderBoxOptions> = {})
     {
-        const { showClass = true, showConfidence = false, showTraceId = false, showNestedClasses = false, target = '$..objects.*' } = options
+        const { showClass = true, showConfidence = false, showTraceId = false, showNestedClasses = false, target = '$..objects.*', boxType = "rich" } = options
```
Comment (tschulz) [inline src/eyepop-render-2d/render-box.ts:28]: We should define the `boxType` as enum

File: src/eyepop-render-2d/index.ts [hunk 2]
```diff
@@ -15,7 +15,7 @@ import { RenderText, RenderTextOptions } from './render-text';
 
 export interface Renderer extends RenderTarget
 {
-    draw(p: Prediction): void
+    draw(p: Prediction, color?:string): void
```
Comment (tschulz) [inline src/eyepop-render-2d/index.ts:18]: Typescript has a type ` @types/color` - should we use that instead here and in styles?

**Review by tschulz — CHANGES_REQUESTED** [review]

File: src/eyepop-render-2d/index.ts (hunk 2 above)
Comment (64blit) [reply src/eyepop-render-2d/index.ts:18]: Probably yes but will come back to this

File: src/eyepop-render-2d/render-box.ts (hunk 1 above)
Comment (64blit) [reply src/eyepop-render-2d/render-box.ts]: Added 👍

Comment (64blit) [reply src/eyepop-render-2d/render-box.ts]: image

**Review by tschulz — APPROVED** [review]

**Pull request merged**